### ISSUE TITLE
Interlinear display size and back/forward buttons

### ISSUE THREAD

**Issue by kuzunguka** [mimbrown/render#111]
Do we want back/forward arrow buttons in the interlinear display like we have in the idiomatic display? And can the interlinear display be shrunk to take up only the required vertical space, as with footnotes?

**Comment by mnjames** [comment]
The second half of @kuzunguka's comment has been split off into issue #113.

I personally don't like the idea of arrows in the interlinear display. I see the fact that you can swipe left and right within that view as an "undocumented feature" and not something we would want to emphasize.

**Comment by socvid** [comment]
I agree with @mnjames for all of his stated reasons and I wouldn't want the buttons taking up reading space in a 3/4s view.

**Comment by kuzunguka** [comment]
My preference is not strong.  And also I was viewing on a large screen, so I can see that left/right buttons become more annoying and less of a feature on a small screen.
